run_linux.go

```diff
@@ -874,9 +875,14 @@ func runUsingRuntime(options RunOptions, configureNetwork bool, moreCreateArgs [
 		}
 	}
 
+	runtimeArgs := options.Args[:]
```

Comment:
nit,question: Is reason for re-slice here is to make copy of `options.Args` or could we just do `runtimeArgs := options.Args`

Reply:
yes, I don't want to modify the input